file_handling.go

```diff
@@ -0,0 +1,237 @@
+package main
+
+import (
+	"bufio"
+	"encoding/csv"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+func loadMembershipDetailsFromCsv(path string) (members map[string]member, err error) {
```

Comment:
Check out https://github.com/gocarina/gocsv - it unmarshals CSV files to a struct (or an array of structs) similar to what you're manually doing here.

Reply:
I've introduced gocsv for all this stuff now. Turned out to be a much bigger job than I anticipated.